pkg/controller/restore-operator/sync.go

```diff
@@ -134,9 +134,10 @@ func (r *Restore) prepareSeed(er *api.EtcdRestore) (err error) {
 	}()
 
 	cs := er.Spec.ClusterSpec
-	clusterName := er.Spec.BackupSpec.ClusterName
+	restoreName := er.Name
```

Comment:
```Go
// Use the restore CR's name as the name of the etcd cluster being restored
clusterName := er.Name
```

Reply:
Can you also update the `EtcdRestore` comment: https://github.com/coreos/etcd-operator/blob/master/pkg/apis/etcd/v1beta2/restore_types.go#L40 ?